.github/workflows/build-plugin-archive.yml

```
@@ -43,6 +43,11 @@ on:
         required: false
         default: 'index.php'
         type: string
+      PLUGIN_DIR_NAME:
+        description: The name of the plugin directory inside the archive. Falls back to the archive name and repository name.
```

Same as above. Also, let's use "folder" to prevent any confusion with "directory" as in "library" or "listing".
```suggestion
        description: The name of the plugin folder (falls back to the archive name, if set, or the repository name).
```